hooks/post_gen_project.py

```
@@ -0,0 +1,154 @@
+
+## How to Use
+
+Add files to this directory that contain important information about your project that you want
+to persist across sessions. These files will be loaded into the AI's context when you start a new session.
+""")
+        print("Created memory-bank/README.md")
+
+    # Remove default-mode if not selected
+    include_default_mode = '{{ cookiecutter.include_default_mode }}' == 'yes'
+    if not include_default_mode:
+        if os.path.exists('roo_config/default-mode'):
```

Ruff (0.8.2)

Use a single `if` statement instead of nested `if` statements

Combine `if` statements using `and`

(SIM102)